server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/cs/configuration/DeployedStoreConfigurationBuilder.java

```diff
@@ -43,4 +43,9 @@ public DeployedStoreConfigurationBuilder customStoreClassName(String customStore
       this.customStoreClassName = customStoreClassName;
       return this;
    }
+
+   @Override
+   public void validate() {
+      super.validate(true);
```

Comment:
I need some context here, I don't see how this could help with `DummyInMemoryStore`...